app/src/main/res/layout/profile_edit_activity.xml

```diff
@@ -12,228 +12,236 @@
       type="org.oppia.android.app.settings.profile.ProfileEditViewModel" />
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/profileEditBackground"
-    android:clipToPadding="false">
+    android:layout_margin="0dp"
```

Comment:
Isn't margin 0dp by default? Why is this needed?

Reply:
Removed.